src/cryptography/convert.js

```diff
@@ -44,7 +44,12 @@ export const getFirstEightBytesReversed = publicKeyBytes =>
 		.slice(0, 8)
 		.reverse();
 
-export const toAddress = buffer => `${bufferToBigNumberString(buffer)}L`;
+export const toAddress = buffer => {
+	const firstEightBytes = Buffer.from(buffer).slice(0, 8);
+	if (firstEightBytes.length < buffer.length)
```

Comment:
Why not just check the length directly?
```
if (Buffer.from(buffer).length > 8)
```
or even
```
if (buffer.length > 8)
```

Reply:
I thought having the Buffer.from(buffer).slice(0, 8) in there will make sure that the input is a proper Buffer.